source/nanoFramework.Hardware.Esp32/TouchPad/TouchPad.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace nanoFramework.Hardware.Esp32.TouchPad
+{
+/// <summary>
+/// TouchPad sensor using interrupts
+/// </summary>
+  public sealed class TouchPad : TouchPadBase
+  {
+    public delegate void TouchPadValueChangedEventHandler(
+        Object sender);
+		//TouchPadValueChangedEventArgs e);
+
+
+		/// <summary>
+		/// Simplified constructor which use default settings
+		/// </summary>
+		/// <param name="pinNumber">Valid touch pads pin number</param>
+		public TouchPad(int pinNumber) : this(pinNumber, new TouchPadConfig())
+		{
+		}
+
+		/// <summary>
+		/// Constructor that allow fine tuning of pit settings
+		/// </summary>
+		/// <param name="pinNumber">Valid touch pads pin number</param>
+		/// <param name="config">Configuration settings</param>
+		public TouchPad(int pinNumber, TouchPadConfig config) : base(pinNumber, config)
+    {
+			SetTouchPadTriggerThreshold(config.InterruptThresholdValue);
+		}
+
+		/// <summary>
+		/// The default FSM mode is ‘TOUCH_FSM_MODE_SW’. If you want to use interrupt trigger mode, 
```

Comment:
When referencing other types (which is great!) you should use the notation `<see cref="TYPE-HERE"/>`.